spring-integration-file/src/main/java/org/springframework/integration/file/config/FileParserUtils.java

```diff
@@ -118,7 +124,7 @@ static void configureFilter(BeanDefinitionBuilder synchronizerBuilder, Element e
 			}
 			if (count != 1) {
 				parserContext.getReaderContext().error("at most one of 'filename-pattern', " +
-						"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
+								"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
```

Comment:
In STS, Ctrl-Shift-F (with nothing selected) reformats the whole file, but if you just select the lines you are interested in it only reformats the selected lines. I try to avoid Ctrl-Shift-F without making a selection first.

I wouldn't mind (although as you know I dislike unnecessary formatting), but in this case, it pushed it past 120, which is bad.